doc/visual-programming/source/conf.py

```diff
@@ -40,13 +40,12 @@
     "sphinx.ext.imgmath",
     "sphinx.ext.ifconfig",
     "sphinx.ext.viewcode",
+    "recommonmark",
 ]
 
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ["_templates"]
 
-source_parsers = {".md": CommonMarkParser}
```

Comment:
You can probably also remove the import on L18

Reply:
Thanks!